pennylane/fermi/conversion.py

```diff
@@ -34,8 +34,15 @@ def jordan_wigner(
 
     .. math::
 
-        a^{\dagger}_N = Z_0 \otimes  ... \otimes Z_{N-1} \otimes \left ( \frac{X-iY}{2} \right ),
-        a_N = Z_0 \otimes  ... \otimes Z_{N-1} \otimes \left ( \frac{X+iY}{2} \right ),
+        a^{\dagger}_0 =  \left (\frac{X_0 - iY_0}{2}  \right ), \:\: \text{...,} \:\:
+        a^{\dagger}_n = Z_0 \otimes Z_1 \otimes ... \otimes Z_{n-1} \otimes \left (\frac{X_n - iY_n}{2} \right ),
+
+    and
```

Comment:
Can the `args` in this method below can now be `~.FermiWord` and `~.FermiSentence` to make them clickable in the docs, or would that not work? 🤔

Reply:
Which method?

Reply:
It looks to me like in the rendered docs they are already clickable. I think Sphinx does some magic when the are in the args section.